TEditXna/Terraria/World.FileV2.cs

```diff
@@ -20,7 +20,7 @@ namespace TEditXNA.Terraria
     public partial class World
     {
         public static uint CompatibleVersion = 102;

```

Comment:
I didn't do that yet because the save portion of the code needs some work
first, will push it this weekend
On Sep 18, 2015 2:15 PM, "BinaryConstruct" <notifications@github.com> wrote:

> In TEditXna/Terraria/World.FileV2.cs
> <https://github.com/TEdit/Terraria-Map-Editor/pull/684#discussion_r39896107>
> :
>
> > @@ -20,7 +20,7 @@ namespace TEditXNA.Terraria
> >      public partial class World
> >      {
> >          public static uint CompatibleVersion = 102;
>
> Can you also update the CompatibleVersion to the latest Terraria map
> version?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TEdit/Terraria-Map-Editor/pull/684/files#r39896107>.
>